roles/st2chatops/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+# st2chatops version to install. Use `latest` to get automatic updates or pin it to numeric version like `2.2.0`
+st2chatops_version: latest
+
+# Please provide ST2_API_KEY using "st2 apikey create -k"
+st2chatops_ST2_API_KEY: "CHANGE-ME-PLEASEdd"
+
+# Hubot Adapter to be used for st2chatops
+st2chatops_HUBOT_ADAPTER: CHANGE-ME-PLEASE
```

Comment:
Just an empty string or None (which won't set the val in `st2chatops.env`) as a default would be nice.

Also I'm more in favor of lowercase vars like: `st2chatops_adapter` or if you want `st2chatops_hubot_adapter`.